modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/CustomizedTableFDSView.java

```diff
@@ -108,18 +114,32 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 
 							moduleName =
 								"default from " + fdsCellRendererCET.getURL();
+
+							clientExtension = true;
 						}
 					}
 				}
 
 				// Use the built-in AMD provided sample as a last resort
 
 				if (moduleName == null) {
-					moduleName = _npmResolver.resolveModuleName(
-						"@liferay/frontend-data-set-sample-web/js" +
-							"/GreenCheckDataRenderer");
+					ServiceContext serviceContext =
```

Comment:
This bit of code is ugly, but since it is part of a sample test module and we will be able to remove it in the future, I'm leaving it like this since we need the `HttpServletRequest` to compose JS URLs and the FDS framework doesn't provide it.